packages/test-lib/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@fuels/test-lib",
```

Comment:
Maybe a better name: `@fuels/jest`

Can this be used for just node projects also? If not, we can prefix it with `react`.

Reply:
yes, it can be used for node also